Build error in benches

```
   Compiling lindera v0.5.1 (/Users/johtani/IdeaProjects/rust-workspace/lindera-workspace/lindera/lindera)
error[E0599]: no function or associated item named `default_normal` found for struct `lindera::tokenizer::Tokenizer` in the current scope
 --> lindera/benches/bench.rs:8:40
  |
8 |         let mut tokenizer = Tokenizer::default_normal();
  |                                        ^^^^^^^^^^^^^^ function or associated item not found in `lindera::tokenizer::Tokenizer`

error: aborting due to previous error

For more information about this error, try `rustc --explain E0599`.
error: could not compile `lindera`.
```

--- a/lindera/benches/bench.rs
+++ b/lindera/benches/bench.rs
@@ -5,7 +5,7 @@ use lindera::tokenizer::Tokenizer;
 
 fn bench_tokenize(c: &mut Criterion) {
     c.bench_function("bench-wiki", |b| {
-        let mut tokenizer = Tokenizer::default_normal();
+        let mut tokenizer = Tokenizer::new("normal", "");
         b.iter(|| tokenizer.tokenize("検索エンジン（けんさくエンジン、英語: search engine）は、狭義にはインターネットに存在する情報（ウェブページ、ウェブサイト、画像ファイル、ネットニュースなど）を検索する機能およびそのプログラム。"))
     });
 }

